share/gpodder/extensions/subscription_stats.py

```diff
@@ -0,0 +1,186 @@
+# 2023/11/24 - Nuno Dias <Nuno.Dias+gpodder@gmail.com>
+# Add Last Fedd and Episode updates
+
+import time
+from time import strftime, localtime
+
+import datetime
+from datetime import datetime
+
+import gpodder
+
+import gi  # isort:skip
+gi.require_version('Gtk', '3.0')  # isort:skip
+from gi.repository import Gtk, Pango  # isort:skip
+
+_ = gpodder.gettext
+
+__title__ = _('Subscription Statistics')
+__description__ = _('Show publishing statistics for subscriptions.')
+__only_for__ = 'gtk'
+__authors__ = 'Brand Huntsman <http://qzx.com/mail/>'
+
+
+class gPodderExtension:
+    def __init__(self, container):
+        self.container = container
+
+    def on_ui_object_available(self, name, ui_object):
+        if name == 'gpodder-gtk':
+            self.gpodder = ui_object
+
+    def on_create_menu(self):
+        # extras menu
+        return [(_("Subscription Statistics"), self.open_dialog)]
+
+    def add_page(self, notebook, type, channels):
+        scrolled = Gtk.ScrolledWindow()
+        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+
+        store = Gtk.ListStore(str, str, str, str)
+        for average, name, last, feed_last, paused in channels:
+            store.append([
+                ('%.1f' % round(average, 1)) if average > 0 else '?', 
+                name if not paused else (_('%s (paused)') % name), last, feed_last
+            ])
+
+        tree = Gtk.TreeView(model=store)
+        scrolled.add(tree)
+
+        dayscell = Gtk.CellRendererText()
+        dayscell.set_property('xalign', 1)
+        dayscell.set_property('alignment', Pango.Alignment.RIGHT)
+        dayscolumn = Gtk.TreeViewColumn(_('Days'))
+        dayscolumn.pack_start(dayscell, True)
+        dayscolumn.add_attribute(dayscell, 'text', 0)
+        tree.append_column(dayscolumn)
+
+        channelcell = Gtk.CellRendererText()
+        channelcell.set_property('xalign', 0)
+        channelcell.set_property('alignment', Pango.Alignment.LEFT)
+        channelcolumn = Gtk.TreeViewColumn(_('Podcast'))
+        channelcolumn.pack_start(channelcell, True)
+        channelcolumn.add_attribute(channelcell, 'text', 1)
+        tree.append_column(channelcolumn)
+
+        lastcell = Gtk.CellRendererText()
+        lastcell.set_property('xalign', 1)
+        lastcell.set_property('alignment', Pango.Alignment.LEFT)
+        lastcolumn = Gtk.TreeViewColumn(_('Last Updated'))
+        lastcolumn.pack_start(lastcell, True)
+        lastcolumn.add_attribute(lastcell, 'text', 2)
+        tree.append_column(lastcolumn)
+
+        channellastcell = Gtk.CellRendererText()
+        channellastcell.set_property('xalign', 0)
+        channellastcell.set_property('alignment', Pango.Alignment.LEFT)
+        channellastcolumn = Gtk.TreeViewColumn(_('Last Feed Updated'))
+        channellastcolumn.pack_start(channellastcell, True)
+        channellastcolumn.add_attribute(channellastcell, 'text', 3)
+        tree.append_column(channellastcolumn)
```

Comment:
Does this column add much information compared to the date of the last episode? It's also empty for feeds fetched with the youtube-dl extension.

Reply:
Maybe you are right, I just put this column because there are some feeds that have recent dates compared with the last episode date, but like you wrote some feeds don't have this date that's the reason they are empty, after some thoughts I will remove this column.